optuna/samplers/random.py

```diff
@@ -34,8 +34,8 @@ class RandomSampler(BaseSampler):
     def __init__(self, seed=None):
         # type: (Optional[int]) -> None
 
-        self.seed = seed
-        self.rng = numpy.random.RandomState(seed)
+        self._seed = seed
```

Comment:
[nits] I think that this field can be removed because it isn't actually used (this point isn't relevant to the main purpose of this PR though).

Reply:
Thanks!